web-animations-1/Overview.bs

```diff
@@ -2296,12 +2311,14 @@ definitions:
 
 An <a>animation effect</a> is in the <dfn>before phase</dfn> if the
 animation effect's <a>local time</a> is not <a>unresolved</a> and
-<em>either</em> of the following conditions are met:
+<em>any</em> of the following conditions are met:
 
 1.  the <a>local time</a> is less than the <a>before-active boundary time</a>,
     <em>or</em>
 1.  the <a>animation direction</a> is &lsquo;backwards&rsquo; and the <a>local
     time</a> is equal to the <a>before-active boundary time</a>.
+1.  the <a>animation timeline</a> <a>phase</a> corresponding to this effect
+    is "before"
```

Comment:
This should use the ["associated with a timeline"](https://drafts.csswg.org/web-animations-1/#associated-with-a-timeline) concept. It needs a period too. And the previous bullet point needs to drop the period and add `, <em>or</em>`.